app-admin/keepassxc/patches/keepassxc-2.1.0.patchset

```diff
@@ -39,5 +39,75 @@ index a77f80a..a688f0a 100644
  if(APPLE)
    install(FILES macosx/keepassxc.icns DESTINATION ${DATA_INSTALL_DIR})
 -- 
-2.2.2
+2.13.1
+
+
+From 30c2ae94a91e1945f15ebfe3ffcbc10c65f7bc7c Mon Sep 17 00:00:00 2001
+From: Chris Moore <chris@mooreonline.org>
+Date: Thu, 3 Aug 2017 16:45:41 +0100
+Subject: Fix invalid parameter types
+
+
+diff --git a/src/format/KeePass2XmlReader.cpp b/src/format/KeePass2XmlReader.cpp
+index dfb03bd..5194451 100644
+--- a/src/format/KeePass2XmlReader.cpp
++++ b/src/format/KeePass2XmlReader.cpp
+@@ -388,7 +388,7 @@ void KeePass2XmlReader::parseBinaries()
+             QString id = attr.value("ID").toString();
+ 
+             QByteArray data;
+-            if (attr.value("Compressed").compare("True", Qt::CaseInsensitive) == 0) {
++            if (attr.value("Compressed").compare(QString("True"), Qt::CaseInsensitive) == 0) {
+                 data = readCompressedBinary();
+             }
+             else {
+-- 
+2.13.1
+
+
+From 84feadaafef8a5568e0977b4dd7bf712ad94015f Mon Sep 17 00:00:00 2001
+From: Chris Moore <chris@mooreonline.org>
+Date: Thu, 3 Aug 2017 17:01:01 +0100
+Subject: Disable unit tests - these fail to build under Qt5 on x86_64
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 916a27c..5135376 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -29,7 +29,7 @@ include(CheckCCompilerFlag)
+ include(CheckCXXCompilerFlag)
+ include(CheckCXXSourceCompiles)
+ 
+-option(WITH_TESTS "Enable building of unit tests" ON)
++option(WITH_TESTS "Enable building of unit tests" OFF)
```

Comment:
This is wrong. Disabling unit tests can and should be done in the recipe itself.
See line 77: `-DWITH_TESTS=on`.